print/layouts/default.vue

```diff
@@ -12,26 +12,95 @@
 export default {
   data() {
     return {
-      clipped: false,
-      drawer: false,
-      fixed: false,
-      items: [
-        {
-          icon: 'mdi-apps',
-          title: 'Welcome',
-          to: '/',
-        },
+      pagedjs: false,
+    }
+  },
+
+  head() {
+    const header = {}
+
+    header.htmlAttrs = {
+      moznomarginboxes: true,
+      mozdisallowselectionprint: true,
+    }
+
+    /**
+     * Define default footer & header
+     * This can be overridden in route views
+     */
+    header.__dangerouslyDisableSanitizersByTagID = {
+      defaultMarginBox: ['cssText'], // disable sanitzing of below inline css
+    }
+
+    const cssPageCounter = `'${this.$tc(
+      'global.margin.pageCounter.page'
+    )} ' counter(page) ' ${this.$tc(
+      'global.margin.pageCounter.of'
+    )}  ' counter(pages)`
+
+    header.style = [
+      {
+        type: 'text/css',
+        hid: 'defaultMarginBox',
+        cssText: `@media print {
+                    
+                    :root {
+                      --ecamp-margin-font-size: 10pt;
+                    }
+
+                    @page {
+                      font-family: "Roboto", sans-serif;
+                      
+                      @top-center {
+                        content: 'eCamp3';
+                        font-size: var(--ecamp-margin-font-size);
+                      }
+                      @bottom-center {
+                        content: ${cssPageCounter};
+                        font-size: var(--ecamp-margin-font-size);
+                      }
+                    }
+                  }`,
+      },
+    ]
+
+    header.script = []
+
+    // inject FRONTEND_URL to client
```

Comment:
Comment is in the wrong place or outdated

Reply:
Seems good to me. The lines 70-77 are responsible to inject FRONTEND_URL to client side